### ISSUE TITLE
Is the library backed up as part of the settings?

### ISSUE THREAD

**Issue by munael** [tachiyomiorg/tachiyomi#8562]
### Steps to reproduce

1. Have a library.
2. Use an app backup system, like Samsung Cloud.
3. Restore apps + app settings on a new/wiped phone.

### Expected behavior

According to #195 (quite old), there's a version of the current library stored as app settings. Are those synched as expected?

### Actual behavior

I don't know. I'm in a situation where data loss is possible and I can't access most info on my device. Including the version.

### Tachiyomi version

0.0.0

### Android version

Android 12. OneUI 4.0.

### Device

Samsung Galaxy A52

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
You tell me, it's not like we work on Samsung Cloud.

**Comment by munael** [comment]
My question is specifically if it's included as part of the settings.

**Comment by arkon** [comment]
I'm not sure that question actually makes sense.

Settings are not registered with the system. Samsung Cloud is not a standard AOSP feature.

I have no idea what Samsung Cloud is doing under the hood. You'd have to try it and find out or point us to developer documentation if you want us to implement some sort of Samsung-specific thing.

**Comment by munael** [comment]
Disregarding the Samsung Cloud example for a bit.

Both backup systems I've interacted with before (the other being Google) would "back up app settings".

Such settings have to be associated with the app in a way that the OS recognizes, unlike the backup proto.gz files. That's what I'm referring to.

Sounds like the probable answer is that it's gone, though...

**Comment by arkon** [comment]
No, the app is opted out from the Google backup system, it had too many issues with it trying to restore too-old data that apps seemingly can't control.
